homeassistant/components/aemet/weather_update_coordinator.py

```diff
@@ -98,6 +98,8 @@ def format_float(value) -> float:
     """Try converting string to float."""
     try:
         return float(value)
+    except TypeError:
+        return None
```

Comment:
If we handle multiple exceptions the same way we can have a single `except` block for all of them inside a tuple.

Reply:
> If we handle multiple exceptions the same way we can have a single `except` block for all of them inside a tuple.

@MartinHjelmare should be fixed now 🙂